website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,71 +1,101 @@
 
-## Component availability
+- It forces engineers to create custom classes and styles to override Helios styles. This can have unexpected long-term effects within the [CSS cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#:~:text=The%20cascade%20is%20an%20algorithm,a%20property%20on%20an%20element).
+- In addition to being difficult to scale, overrides nudge the visual language of your product out of sync with Helios and with the rest of the HashiCorp product line.
 
-We are in the process of sunsetting [Structure](https://github.com/hashicorp/structure). As we work to achieve parity between Structure and the Helios Design System, use this decision tree to understand how to move forward within your project:
+### Resizing components
 
-![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
+Helios components are designed and built to layout-agnostic, meaning the task of laying out components in a design is left to the consumer.
+
+Unless specifically mentioned in the documentation, the component in code will fill the parent container it is used in. For designers, it's helpful to use [auto layout](https://help.figma.com/hc/en-us/articles/5731482952599-Using-auto-layout#:~:text=You%20can%20add%20auto%20layout%20to%20a%20selected%20frame%2C%20component,and%20select%20Add%20Auto%20layout) wherever possible which closely mimics [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and other layout mechanisms in the browser. Helios Figma components are flexible and can be set to `fill container` within an auto layout container, which replicates the experience of a development environment.
 
-**Does the element (or a comparible alternative) exist in Helios?**
+!!! Warning
 
-If so, great! Using this element ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
+**Fixed width components**
 
-**If no, does the element (or a comparible alternative) exist in Structure?**
+Setting a component to a fixed width in Figma is akin to setting an explicit `width` value on a component or element in code. This generally goes against fluid and responsive best practices and should be avoided in Figma. Instead, use auto layout wherever possible.
+!!!
 
-If yes, use this component with the understanding that:
+## Working with foundations (styles)
 
-1. Structure will be deprecated/sunsetted in the future
-2. Not all Figma components are built in code
-3. There is no longer a support structure in place if errors/bugs arise
+Similar to components, using styles defined in the Helios [Product Foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?t=4kdgl88SMIiEYhbA-1) reinforces consistency and can help to resolve problems with style usage and implementation. Detaching from Helios styles is not recommended in most circumstances.
```

Comment:
Trying to reinforce that "foundations" and "styles" are somewhat interchangeable in our language. And a minor tweak to use a more active voice on the last sentence.

```suggestion
Similar to components, using the foundational styles defined in the Helios [Product Foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?t=4kdgl88SMIiEYhbA-1) library reinforces consistency and can help to resolve problems with style usage and implementation. We don't recommend detaching from Helios styles in most circumstances.
```